src/txn/txn.c

```diff
@@ -1654,6 +1654,7 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
 #endif
     prepare = F_ISSET(txn, WT_TXN_PREPARE);
     readonly = txn->mod_count == 0;
+    save_errors = F_ISSET(session, WT_SESSION_SAVE_ERRORS);
```

Comment:
Are there any cases where `save_errors` is false here? I would be worried it if there is a case like that.